web/static/js/components/folders/FolderShow.jsx

```diff
@@ -28,14 +30,16 @@ class FolderShow extends Component<any, any> {
     totalCount: PropTypes.number.isRequired,
     respondentsStats: PropTypes.object.isRequired,
     params: PropTypes.object,
-    id: PropTypes.number,
+    folderId: PropTypes.number,
```

Comment:
If now there is a `surveyFolder`, couldn't we remove this prop and when necessary just use `surveyFolder.id` ?

Reply:
Yeah, probably! I'll try it! Thanks!